lib/semian/protected_resource.rb

```diff
@@ -1,8 +1,10 @@
+require 'forwardable'
+
 module Semian
   class ProtectedResource
```

Comment:
Just to recap, a protected resource is a resource that's both behind Semian and a Circuit Break?